cpp/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
 cmake_minimum_required(VERSION 3.10.0)
 
-project(Mikado-TestBuilders-Workshop CXX)
```

Comment:
Are you sure naming a CMake global "ProjectId" is following the CMake conventions? I see a CMAKE_TOOLCHAIN_FILE just below?

Reply:
I change this line, however, I am not sure of the coventions! I need to look this up.